trappy/plotter/__init__.py

```diff
@@ -70,7 +70,10 @@ def plot_trace(trace,
         raise RuntimeError("plot_trace needs ipython environment")
 
     if not isinstance(trace, trappy.BareTrace):
-        trace = trappy.FTrace(trace)
+        if trace.endswith("html"):
+            trace = trappy.SysTrace(trace)
+        else:
+            trace = trappy.FTrace(trace)
```

Comment:
I don't like having to rely on file names but in this case I don't think we have an alternative so let's do it :shipit: